classwrap.d.ts

```diff
@@ -0,0 +1,17 @@
+export as namespace classwrap
+
+export interface ClassSet {
+  [key: string]: boolean | any | ClassSet
+}
+
+export type Class = string | number | ClassSet
+
+/**
+ * Joins all elements of an array or keys of an object into a string.
+ * If the value associated with a given key is falsy, the key will be ignored.
+ * Nested arrays or objects are supported too. Use this feature to assemble
+ * classes with a common prefix.
+ *
+ * @param classes
+*/
+export default function wrap(classes: Class | Class[])
```

Comment:
I have longley hesitated about define a `Classes` type or a `Class` type.

But here I chosen to define a `Class` type and to declare that the wrap function can take either a `Class` or an array of `Class`

Reply:
Sounds good!